syntax = "proto3";

package ocr;

service service_realsence{
    rpc OcrImage(RequestImage) returns (ReplyCharacters) {}

}

message RequestImage{
    int32 width = 1;
    int32 height = 2;
    bytes image_data = 3;
}

message ReplyCharacters{
 repeated string characters = 1;
}